.github/workflows/ci.yml

```diff
@@ -223,6 +223,7 @@ jobs:
     permissions:
       id-token: write # This is required for requesting the JWT
     steps:
+      - uses: ilammy/setup-nasm@v1
```

Comment:
We'll need to install this on the Jenkins machine

Reply:
yep. i will handle that